grails-app/domain/mama/ng/scheduler/User.groovy

```diff
@@ -22,4 +22,8 @@ class User {
         GENERAL,
         SUPER
     }
+
+    boolean checkPassword(String password) {
+        return (password.bytes.encodeBase64().toString().equals(passwordHash))
```

Comment:
The password is hashed before it's stored. Due to time constraints I will leave this as is for now.